src/replies/ru.json

```
@@ -0,0 +1,34 @@
+{
+	"wporg-1": {
+		"label": "Вопрос в чужой теме",
+		"content": "Не нужно задавать свои вопросы в чужих темах, согласно <a href=\"https://ru.wordpress.org/support/forum-rules/\">Правилам форума (п2)</a> следует создать отдельную тему со своим вопросом."
+	},
+	"wporg-2": {
+			"label": "Вопрос по коммерческому продукту",
+			"content": "Поддержка платных плагинов и тем оказывается продавцом или автором, см <a href=\"https://ru.wordpress.org/support/forum-rules/\">Правила форума (п9)</a>. Покупая платный плагин или тему вы также оплачиваете и стоимость поддержки. Если срок платной подписки истек - его следует продлить, с продлением вы также получите и обновления с исправлением накопившихся ошибок."
+	},
```

```suggestion
	"wporg-2": {
		"label": "Вопрос по коммерческому продукту",
		"content": "Поддержка платных плагинов и тем оказывается продавцом или автором, см <a href=\"https://ru.wordpress.org/support/forum-rules/\">Правила форума (п9)</a>. Покупая платный плагин или тему вы также оплачиваете и стоимость поддержки. Если срок платной подписки истек - его следует продлить, с продлением вы также получите и обновления с исправлением накопившихся ошибок."
	},
```

Fixes an extra indent that was included.